packages/twenty-front/src/utils/date-utils.ts

```diff
@@ -212,3 +214,20 @@ export const formatToHumanReadableDateTime = (date: Date | string) => {
     minute: 'numeric',
   }).format(parsedJSDate);
 };
+
+export const getDateFormatString = (
```

Comment:
neat: I'm a bit confused why we have utils here and in ui/input/components/internal/date